src/Models/Constants/Interfaces.ts

```diff
@@ -756,6 +756,7 @@ export interface IADT3DViewerProps extends BaseComponentProps {
      * Call to provide customized styling that will layer on top of the variant rules.
      */
     styles?: IStyleFunctionOrObject<IADT3DViewerStyleProps, IADT3DViewerStyles>;
+    selectedLayers?: string[];
```

Comment:
move this above styles so it's still alphabetized.

Reply:
Done